pkg/reconcile/pipeline/infinispan/handler/manage/statefulset_updates.go

```diff
@@ -160,8 +160,7 @@ func StatefulSetRollingUpgrade(i *ispnv1.Infinispan, ctx pipeline.Context) {
 	}
 
 	// Validate extra Java options changes
-	if updateStatefulSetEnv(container, statefulSet, "EXTRA_JAVA_OPTIONS", ispnContr.ExtraJvmOpts) {
```

Comment:
I think we can remove the `EXTRA_JAVA_OPTIONS` env variable entirely. AFAICT it's never used by the image or server logic.